recipe/conda_build_config.yaml

```diff
@@ -356,6 +359,8 @@ cairo:
   - 1.16
 cfitsio:
   - 3.470
+cudnn:
+  - 7
```

Comment:
If the min pin is `x.x` do we need the pinned version to `x.x`?

Reply:
Well this is more-or-less the same discussion we are having in [this thread]( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/309#pullrequestreview-305533073 ). Namely do we want a policy of tightly pinning the minimum or not?

With `cudnn` I would suspect users, developers, and maintainers are eager to get the latest version in their builds for the latest optimizations. So kept this loose.

With `nccl` I'm guessing users are more concerned that things stay operational.

I could be wrong about both of these guesses. As a community we may decide that other things are more important than our thoughts on user preferences.

Kind of a side note, but the default `min_pin` is `x.x.x.x.x.x.x`.